src/desktop/apps/auction/components/layout/active_bids/__tests__/MyActiveBids.jest.js

```diff
@@ -5,7 +5,7 @@ const { Auction } = require("desktop/models/auction")
 const CurrentUser = require("desktop/models/current_user")
 
 // FIXME: Add required props
-xdescribe("apps/auction/components/layout/active_bids/MyActiveBids", () => {
+describe.skip("apps/auction/components/layout/active_bids/MyActiveBids", () => {
```

Comment:
xdescribe, xit should be changed to describe.skip and it.skip

Reply:
weird, is that a jest thing? they stopped supporting xit?

Reply:
[they still support](https://jestjs.io/docs/api#describeskipname-fn) but now jest breaks complaining to use skip instead, so the fix was to change instead of suppressing the warning